external/js/cothority/src/calypso/keccak.d.ts

```diff
@@ -0,0 +1 @@
+declare module "keccak";
```

Comment:
this file should be removed

Reply:
if I run `npm run test` without that file, I get
```
TSError: ⨯ Unable to compile TypeScript:
src/darc/darc.ts(67,22): error TS2686: 'Long' refers to a UMD global, but the current file is a module. Consider adding an import instead.
```